<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="colorToMixLabel">
        <property name="text">
         <string>Color To Mix</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="colorToMixView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="backgroundBrush">
         <brush brushstyle="NoBrush">
          <color alpha="0">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
        <property name="foregroundBrush">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
        <property name="interactive">
         <bool>true</bool>
        </property>
        <property name="sceneRect">
         <rectf>
          <x>0.000000000000000</x>
          <y>0.000000000000000</y>
          <width>0.000000000000000</width>
          <height>0.000000000000000</height>
         </rectf>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="colorToMixCombo"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="mixedColorLabel">
        <property name="text">
         <string>Result</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="mixedColorView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="mixColor1Label">
        <property name="text">
         <string>Mix color 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="mixColor1View"/>
      </item>
      <item>
       <widget class="QComboBox" name="mixColor1ComboBox"/>
      </item>
      <item>
       <widget class="QSlider" name="mixColor1Slider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mixColor1SliderLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="mixColor2Label">
        <property name="text">
         <string>Mix color 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="mixColor2View"/>
      </item>
      <item>
       <widget class="QComboBox" name="mixColor2ComboBox"/>
      </item>
      <item>
       <widget class="QSlider" name="mixColor2Slider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mixColor2SliderLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="mixColor3Label">
        <property name="text">
         <string>Mix color 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="mixColor3View"/>
      </item>
      <item>
       <widget class="QComboBox" name="mixColor3ComboBox"/>
      </item>
      <item>
       <widget class="QSlider" name="mixColor3Slider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mixColor3SliderLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="mixColor4Label">
        <property name="text">
         <string>Mix color 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="mixColor4View"/>
      </item>
      <item>
       <widget class="QComboBox" name="mixColor4ComboBox"/>
      </item>
      <item>
       <widget class="QSlider" name="mixColor4Slider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mixColor4SliderLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>colorToMixCombo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>SetColourToMix(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixColor1ComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>SetColour1(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixColor2ComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>SetColour2(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixColor3ComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>SetColour3(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixColor4ComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>SetColour4(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>898</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixColor1Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>RecomputeMixedColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixColor2Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>RecomputeMixedColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixColor3Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>RecomputeMixedColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixColor4Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>RecomputeMixedColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>898</x>
     <y>616</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixColor1Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mixColor1SliderLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>667</x>
     <y>833</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>856</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixColor2Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mixColor2SliderLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>932</x>
     <y>833</y>
    </hint>
    <hint type="destinationlabel">
     <x>932</x>
     <y>856</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixColor3Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mixColor3SliderLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1196</x>
     <y>833</y>
    </hint>
    <hint type="destinationlabel">
     <x>1196</x>
     <y>856</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mixColor4Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mixColor4SliderLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1461</x>
     <y>833</y>
    </hint>
    <hint type="destinationlabel">
     <x>1461</x>
     <y>856</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetColourToMix(QString)</slot>
  <slot>SetColour1(QString)</slot>
  <slot>SetColour2(QString)</slot>
  <slot>SetColour3(QString)</slot>
  <slot>SetColour4(QString)</slot>
  <slot>RecomputeMixedColor()</slot>
 </slots>
</ui>
